### PR TITLE
Initial PCSA commit with some Reporting features

### PR DESCRIPTION
**Pull request by Buddhiprabha**
I added the PCSA project skeleton and the Reporting features that I have developed so far. 

Reporting features added are as follows:
- Landing page for reporting
- Reporting Steps
- Reporting Types
- Reporting FAQ
- More Reporting Resources
- Confidentiality
- Post Staff contact - partially done

### CODE REVIEW

File: app/src/main/java/com/pcsa/beans/reporting/LocationDetails.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * @author Buddhiprabha Erabadda
+ *
+ * Holds contact information for a particular location
+ */
+public class LocationDetails {
+    private String locationName;
+    private int pcmo_contact;
+    private int ssm_contact;
+    private int sarl_contact;
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/beans/reporting/LocationDetails.java]: Naming could be more specific, or give a description for later reference

File: app/src/androidTest/java/com/pcsabuddhi/ApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.pcsabuddhi;
```
Comment (shruti-gupta) [inline app/src/androidTest/java/com/pcsabuddhi/ApplicationTest.java]: The name should not contain your name :p

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ * @author Buddhiprabha Erabadda
+ *
+ * Allows the user to call Post Staff in case of crime. The details for the
+ * current location will be set by changing the location
+ */
+public class ContactPostStaff extends Activity implements AdapterView.OnItemSelectedListener {
+
+    Button contact_pcmo;
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: Naming should be done, showing the purpose of the element. Also, java coding standards prefer usage of camel case for variables

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    Button contact_ssm;
+    Button contact_sarl;
+    TextView current_location;
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: same about naming convention

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    private static final Map<String, LocationDetails> locationDetails;
+    static {
+        locationDetails = new HashMap<>();
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: is this some test code? do you mean to change it later?

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_reporting_contact_post_staff);
+
+        contact_pcmo = (Button) findViewById(R.id.post_staff_pcmo);
+        contact_ssm = (Button) findViewById(R.id.post_staff_ssm);
+        contact_sarl = (Button) findViewById(R.id.post_staff_sarl);
+        current_location = (TextView) findViewById(R.id.post_staff_current_location);
+
+        contact_pcmo.setText(R.string.contact_pcmo);
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: why not do it in the xml?

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        contact_ssm.setText(R.string.contact_ssm);
+        contact_sarl.setText(R.string.contact_sarl);
+
+        Spinner locationList = (Spinner) findViewById(R.id.reporting_locationlist);
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: write comments to explain what and why you are doing anything

File: app/src/main/java/com/pcsa/views/reporting/HomeScreen.java [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+import com.pcsabuddhi.R;
+
+/**
+ * @author Buddhiprabha Erabadda
+ *
+ * Landing page of Reporting which has links for Reporting functinalities
+ */
+public class HomeScreen extends Activity {
+
+    Button reporting_steps;
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/HomeScreen.java]: same about naming

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java (hunk 5 above)
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: What are you planning to use for this?

**Comment by rosariorobinson** [comment]
where are we with this one?

**Comment by Buddhiprabha** [comment]
For this the suggested changes were done in a separate PR since Ilo had merged this PR before code review by mistake. You can see that [here](https://github.com/systers/PCSA/pull/3)

**Pull request merged**